components/dashboard/src/data/organizations/orgs-query.ts

```diff
@@ -92,8 +92,7 @@ export function useCurrentOrg(): { data?: OrganizationInfo; isLoading: boolean }
         orgId = orgIdParam;
     }
     let org = orgs.data.find((org) => org.id === orgId);
-    if (!org && user?.additionalData?.isMigratedToTeamOnlyAttribution) {
```

Comment:
@svenefftinge No hard dependency on this code change, but: Did you check the prod DB whether this flag is set for all users?

Reply:
yes